typescript/static-site/static-site.ts

```diff
@@ -70,21 +69,29 @@ export class StaticSite extends Construct {
     });
     new CfnOutput(this, 'Certificate', { value: certificate.certificateArn });
 
+    
     // CloudFront distribution
     const distribution = new cloudfront.Distribution(this, 'SiteDistribution', {
       certificate: certificate,
+      defaultRootObject: "index.html",
       domainNames: [siteDomain],
+      minimumProtocolVersion: cloudfront.SecurityPolicyProtocol.TLS_V1_2_2021,
```

Comment:
This line isn't completely necessary - this value will get set by default if you don't include this prop, but it doesn't hurt I guess.

Reply:
Good point. I'm noticing that it also gets set as a feature flag (for lack of a better term) in cdk.json.  I guess leaving it in would make it more explicit for future readers of this code.